### DISCUSSION TITLE
Change CSS classes

### DISCUSSION THREAD

**Discussion by christiancazzaro** [easepick/easepick#95] [Q&A]
Hi

How can I change the tooltip background-color?

Thank you

**Comment by wakirin** [comment]
https://easepick.com/examples/customize.html
You need to create a CSS file with your style and add this file after easepick styles.

**Reply by chris-eaton** [reply]
Is there a way to style it normally (via CSS) or at the very least add inline styles during startup.

 I am using a bundler that puts all my CSS into a single package.  It makes it hard to break off single CSS files for a single component.

For my specific use case I only want to tweak 1 or 2 styles to help it fit in better, so seems overkill to copy the styles and create a new file & distribution for it.

**Reply by Codebreaker101** [reply]
`css` option can be a `function`, `string` or `array` of links. [Check details here](https://github.com/easepick/easepick/blob/4a49a0f029551aabb9f890d8458b0d7a2046842e/packages/core/src/core.ts#L483). You can do something like this:

``` js
const picker = new easepick.create({
    element: "#datepicker",
    zIndex: 10,
    inline: true,
    grid: 2,
    calendars: 2,
    plugins: ["RangePlugin","PresetPlugin"],
    css: function(s) {
        console.log(s)
        /* to load default style into shadow dom */
        const cssLinks = ["https://cdn.jsdelivr.net/npm/@easepick/bundle@1.2.0/dist/index.css"]
        cssLinks.forEach(cssLink => {
            const link = document.createElement('link');
            link.href = cssLink;
            link.rel = 'stylesheet';
            const onReady = () => {
            this.cssLoaded++;

            if (this.cssLoaded === cssLinks.length) {
                this.ui.wrapper.style.display = '';
            }
            };
            link.addEventListener('load', onReady);
            link.addEventListener('error', onReady);
            this.ui.shadowRoot.append(link);
        })

        /* append custom style css */
        const css = `
            .container.range-plugin {
                box-shadow: none;
            }
        `
        const style = document.createElement('style');
        const styleText = document.createTextNode(css);
        style.appendChild(styleText);

        this.ui.shadowRoot.append(style);
        this.ui.wrapper.style.display = '';
    }
})
```

**Reply by akshay996** [reply]
Thanks @Codebreaker101, your code worked for me!

**Comment by ghost** [comment]
We made it like so
````
            css: [
              "/themes/custom/book/dist/css/easepick.css",
            ],
````
We placed a custom css path into picker's settings. And then we generate this file from a scss file at is in node_modules folder.
In scss we can override variables and customize colors.

![image](https://user-images.githubusercontent.com/88319069/204323519-dfbdddf6-4cec-4ed1-9210-1fa548bb5cbe.png)
